lib/file_items_gateway.rb

```diff
@@ -0,0 +1,23 @@
+require 'json'
+
+class FileItemsGateway
+  FILE_PATH = "#{__dir__}/../tmp/items.json"
+
+  def all
+    return [] unless File.exist?(FILE_PATH)
+    File.open(FILE_PATH, 'r') do |file|
+      items = JSON.parse(file.read, symbolize_names: true)
+      items
+    end
```

Comment:
Does this work with just ```JSON.parse(file.read, symbolize_names: true)``` ?
Or do you have to assign to a items then return?

Reply:
Not sure, we'll definitely give it a go!